### PR TITLE
Amberish runs

### PR DESCRIPTION
**Pull request by epwalsh**
Includes some additions to the model config.

### CODE REVIEW

**Review by soldni — APPROVED** [review]
lgtm!

File: configs/amberish13-weka.yaml [hunk 1]
```diff
@@ -0,0 +1,1293 @@
+run_name: amberish7-run001
```
Comment (2015aroras) [inline configs/amberish13-weka.yaml:1]:
```suggestion
run_name: amberish13-run001
```

File: olmo/config.py [hunk 2]
```diff
@@ -1215,7 +1231,9 @@ def autocast_precision(self) -> torch.dtype:
     @property
     def fsdp_precision(self) -> MixedPrecision:
```
Comment (2015aroras) [inline olmo/config.py:1232]:
```suggestion
    def fsdp_precision(self) -> Optional[MixedPrecision]:
```

File: scripts/beaker/unshard.sh [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+pip install awscli
+pip install '.[train]'
+pip freeze
+
+# Move AWS credentials from env to relevant files
+mkdir -p ~/.aws
+printenv AWS_CONFIG > ~/.aws/config
+printenv AWS_CREDENTIALS > ~/.aws/credentials
+
+checkpoint=ai2-llm/checkpoints/OLMo-medium/amberish7/step477850
```
Comment (2015aroras) [inline scripts/beaker/unshard.sh:19]: This wasn't the purpose of your PR, but by making this a parameter you can make this script easily usable by everyone else

File: scripts/beaker/unshard.sh [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+
+echo "Unsharding..."
+python scripts/unshard.py "/weka/oe-training-default/${checkpoint}" \
+    "/weka/oe-training-default/${checkpoint}-unsharded" \
+    --type=olmo_core
```
Comment (2015aroras) [inline scripts/beaker/unshard.sh:24]: You can probably remove the type

**Review by 2015aroras — APPROVED** [review]

File: scripts/beaker/unshard.sh (hunk 3 above)
Comment (epwalsh) [reply scripts/beaker/unshard.sh]: Done: ba67ac0e

File: scripts/beaker/unshard.sh (hunk 4 above)
Comment (epwalsh) [reply scripts/beaker/unshard.sh]: Done: ba67ac0e

**Pull request merged**